<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForumV2Details</class>
 <widget class="QDialog" name="ForumV2Details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forum Details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/rstray3.png</normaloff>:/images/rstray3.png</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="stabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../images.qrc">
        <normaloff>:/images/info16.png</normaloff>:/images/info16.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Forum Details</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Forum Info</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Forum Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameline"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Popularity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="popline">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Last Post</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="postline">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Forum ID</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="IDline"/>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Forum Description</string>
            </property>
            <layout class="QGridLayout">
             <item row="0" column="0">
              <widget class="QTextEdit" name="DescriptiontextEdit"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite">
      <attribute name="icon">
       <iconset resource="../images.qrc">
        <normaloff>:/images/encrypted22.png</normaloff>:/images/encrypted22.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Security</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Allowed Messages</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_authd">
            <property name="text">
             <string>Authenticated Messages</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_anonymous">
            <property name="text">
             <string>Anonymous Messages</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>358</width>
           <height>172</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
